blueprints/addon/files/tsconfig.json

```
@@ -16,9 +16,9 @@
       "<%= addonName %>/test-support": ["addon-test-support"],
       "<%= addonName %>/test-support/*": ["addon-test-support/*"],
       "*": ["types/*"]
-    }
-  },
-  "types": [
-    "ember-source/types"
-  ]
+    },
+    "types": [
+      "ember-source/types"
```

```suggestion
      "./node_modules/ember-source/types/stable"
```

We need https://github.com/emberjs/ember.js/pull/20792 I think.

---

A new beta was published, so this should be fine now.